<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_selectImage_file</class>
 <widget class="QDialog" name="Dialog_selectImage_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>266</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Selecionar Arquivo de Imagem</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc_file.qrc">
    <normaloff>:/Ariel/img/Ariel/Icon.png</normaloff>:/Ariel/img/Ariel/Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(163,181,191);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>461</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="txt_compatibilityFile">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_compatibilityFile">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222,233,239);
color: rgb(0, 0, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Arquivo de compatibilidade</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_add">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16,6,144);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Adicionar </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16,6,144);
color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>421</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="txt_imageFile">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_addImageFile">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222,233,239);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Arquivo de Imagem</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rsc_file.qrc"/>
 </resources>
 <connections/>
</ui>
